kedro/framework/hooks/specs.py

```diff
@@ -4,6 +4,7 @@
 """
 from typing import Any, Dict, Optional
 
+from kedro.framework.context.context import KedroContext
```

Comment:
```suggestion
from kedro.framework.context import KedroContext
```

Reply:
Ops I think it is actually 
`from kedro.framework.context.context import KedroContext`

https://github.com/kedro-org/kedro/blob/main/kedro/framework/context/context.py